build.sh

```diff
@@ -1049,6 +1049,9 @@ EOM
             sh build.sh verify-$target 2>&1 | tee build/build.log | xcpretty -r junit -o build/reports/junit.xml || failed=1
             if [ "$failed" = "1" ] && cat build/build.log | grep -E 'DTXProxyChannel|DTXChannel|out of date and needs to be rebuilt|operation never finished bootstrapping'; then
                 echo "Known Xcode error detected. Running job again."
+                if cat build/build.log | grep -E 'out of date and needs to be rebuilt'; then
```

Comment:
We do start off with an empty build directory. I don't know what causes the pcm file to appear out of date, but I can say for sure that our current approach of just re-running xcodebuild will never succeed without first deleting the outdated file.